Directory.Build.props

```diff
@@ -151,6 +151,7 @@
 
   <PropertyGroup Label="CalculateOS">
     <!-- Default to portable build if not explicitly set -->
+    <PortableBuild Condition="'$(PortableBuild)' == '' and '$(DotNetBuildFromSource)' == 'true'">false</PortableBuild>
```

Comment:
I've added this as the default when `DotNetBuildFromSource` is set to avoid having to pass more arguments.